src/components/property-tree/property-tree-utils.tsx

```diff
@@ -45,9 +45,7 @@ const generateNode = (name: string, childList: TreeAsset[], id: string): TreeAss
 
 const generatePrinciple = (asset: Asset, childNodes: TreeAsset[]): TreeAsset => {
   return {
-    title: (
-      <span>{asset.assetLocation?.parentAssets?.length ? "Principle" : "Hackney"}</span>
-    ),
+    title: <span>Hackney</span>,
     children: [
```

Comment:
Is principal not required anymore?

Reply:
I don't think so, not sure it was put there to start with. With Sean we confirmed last week that if the asset has no parents, it's displayed under "Hackney". If it has parents then the parents will show, and Hackney wouldn't be displayed at the top.